### ISSUE TITLE
kubeadm preflight-checks on /var/lib/kubelet/

### ISSUE THREAD

**Issue by ReSearchITEng** [kubernetes/kubeadm#482]
## Versions

**kubeadm version** (use `kubeadm version`):
kubeadm version: &version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.0", GitCommit:"6e937839ac04a38cac63e6a7a306c5d035fe7b0a", GitTreeState:"clean", BuildDate:"2017-09-28T22:46:41Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}

**kubelet** `kubelet --version
Kubernetes v1.8.0`

**Environment**:
- **Kubernetes version** (use `kubectl version`):
- **Cloud provider or hardware configuration**:
on prem, using Vagrantfile from: https://github.com/ReSearchITEng/kubeadm-playbook/
- **OS** Centos 7.4

## What happened?

systemctl start kubelet creates key files:
```
[root@k8s-master ~]# ls -la /var/lib/kubelet/pki/kubelet.*
-rw-r--r--. 1 root root 1115 Oct  3 20:59 /var/lib/kubelet/pki/kubelet.crt
-rw-------. 1 root root 1675 Oct  3 20:59 /var/lib/kubelet/pki/kubelet.key
```
triggering errors in pre-flight check. 
On the other side, we don't want to disable all checks.
```
fatal: [k8s-master]: FAILED! => {"changed": true, "cmd": ["/usr/bin/kubeadm", "init", "--config", "/etc/kubernetes/kubeadm-master.conf"], "delta": "0:00:00.922327", "end": "2017-10-03 20:52:07.588033", "failed": true, "msg": "non-zero return code", "rc": 2, "start": "2017-10-03 20:52:06.665706", "stderr": "[preflight] Some fatal errors occurred:\n\t/var/lib/kubelet is not empty\n[preflight] If you know what you are doing, you can skip pre-flight checks with `--skip-preflight-checks`", "stderr_lines": ["[preflight] Some fatal errors occurred:", "\t/var/lib/kubelet is not empty", "[preflight] If you know what you are doing, you can skip pre-flight checks with `--skip-preflight-checks`"], "stdout": "[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.\n[init] Using Kubernetes version: v1.7.7\n[init] Using Authorization modes: [Node RBAC]\n[preflight] Running pre-flight checks", "stdout_lines": ["[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.", "[init] Using Kubernetes version: v1.7.7", "[init] Using Authorization modes: [Node RBAC]", "[preflight] Running pre-flight checks"]}
```

## What you expected to happen?
pre-flight checks skip the /var/lib/kubelet/pki/kubelet.crt and /var/lib/kubelet/pki/kubelet.key files

## How to reproduce it (as minimally and precisely as possible)?
systemctl start kubelet
kubeadm init

**Comment by kad** [comment]
This will be solved once https://github.com/kubernetes/kubernetes/pull/53317 merged.

**Comment by liggitt** [comment]
also discussed in https://github.com/kubernetes/kubernetes/issues/53356#issuecomment-333748618

**Comment by jamiehannaford** [comment]
Fixed by https://github.com/kubernetes/kubernetes/pull/53317 (and cherry-picked in https://github.com/kubernetes/kubernetes/pull/53448). Probably worth waiting until 1.8.1 is cut until we close this out.

**Comment by liggitt** [comment]
fixed in v1.8.1
